tests/queries/0_stateless/02815_analyzer_aggregate_functions_of_group_by_keys.sql

```diff
@@ -0,0 +1,39 @@
+set allow_experimental_analyzer = 1;
```

Comment:
The test requires comments, which will be shown in the reference file as well. So, it'd be much easier to find a query and corresponding output in reference file (and wise versa)